docs/docs/configuration/db.md

```diff
@@ -92,6 +92,20 @@ You can reference the OCI manifest of [trivy-db].
     `trivy-db-registry:latest` => `trivy-db-registry:latest`, but `trivy-db-registry` => `trivy-db-registry:2`.
 
 
+### Rate limits
+Trivy hosts its databases on public OCI registries that are subject to their respective rate limits. While we strive to make the databases available to every
+Trivy user, there are certain recommendations that one can make in order to ensure rate limits are not hit.
+
+#### Authenticated use of Registries
+By authenticating with the registries that Trivy hosts its DBs on can significantly increase the limit for users. For Amazon ECR, the details for rate limits can be found [ecr-limits].
+For GitHub GHCR, the rate limits can be found [ghcr-limits]
```

Comment:
It's not correct. GHCR doesn't respect the GitHub token now.

Reply:
But some users wrote that GH token helps them.
Do you think that for GHCR it makes no difference at all whether the user is authorized or not when downloading the image?

Reply:
>But some users wrote that GH token helps them.

Trivy has [a bug](https://github.com/aquasecurity/trivy/pull/7580) that doesn't reflect `GITHUB_TOKEN` now, so I'm sure those users luckily didn't see rate limit errors.

I saw [the same rate limit error](https://github.com/aquasecurity/trivy/pull/7580#issuecomment-2370624673) when I set the token, so I believe it has no effect on GHCR. If we confirm it helps, we can add it back.

Reply:
Got it!
Removed info about GHCR in https://github.com/aquasecurity/trivy/pull/7679/commits/c18a51169c103016f73c8ce61a86a9c191109fb8